resource/definitions/loader/3dsmax/camera-3dsmax-loader.json

```diff
@@ -0,0 +1,215 @@
+{
+  "type": "loader",
+  "name": "Camera Loader",
+  "asset_type": "cam",
+  "host_type": "3dsmax",
+  "ui_type": "qt",
+  "contexts": [
+    {
+      "name": "main",
+      "stages": [
+        {
+          "name": "context",
+          "plugins":[
+            {
+              "name": "context selector",
+              "plugin": "common_passthrough_loader_context",
+              "widget": "common_passthrough_loader_context"
+            }
+          ]
+        }
+      ]
+    }
+  ],
+  "components": [
+    {
+      "name": "snapshot",
+      "file_formats": [".mb", ".ma"],
+      "stages": [
+        {
+          "name": "collector",
+          "plugins":[
+            {
+              "name": "Collect components from context",
+              "plugin": "common_context_loader_collector"
+            }
+          ]
+        },
+        {
+          "name": "importer",
+          "plugins":[
+            {
+              "name": "Import paths to Max",
+              "plugin": "max_native_loader_importer",
+              "options": {
+                "load_mode": "import",
```

Comment:
this options are from maya, please review comparing to the old 3dmax definitions